monitoring/bin/change_grafana_admin_password.sh

```diff
@@ -0,0 +1,133 @@
+#! /bin/bash
+
+# Copyright © 2022, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cd "$(dirname $BASH_SOURCE)/../.."
+source monitoring/bin/common.sh
+
+this_script=`basename "$0"`
+
+function show_usage {
+  log_message  "Usage: $this_script [--namespace NAMESPACE --tenant TENANT]"
+  log_message  ""
+  log_message  "Changes the password of the Grafana admin user."
+  log_message  ""
+  log_message  "To change the Grafana admin user at the cluster level, do not pass any "
+  log_message  "arguments.  To change the Grafana admin user at the tenant level, you need"
+  log_message  "to provide the following arguments:"
+  log_message  "     -ns, --namespace NAMESPACE   - The namespace where the Viya tenant resides."
+  log_message  "     -t,  --tenant TENANT         - The tenant whose logging data source you want to set up."
```

Comment:
Does this script set up "logging data sources"?  I suspect a cut-and-paste without updating the text completely.

Reply:
It was copy-paste that wasn't updated properly.  It will be updated in the next push.